setup.py

```diff
@@ -14,6 +15,9 @@
     else:
         reqs += [ir]
 
+if os.environ.get('INSTALL_MPI') == '1':
```

Comment:
you probably meant MPI_INSTALL ...

but this is probably not the way we should do it; I don't think ``setup.py`` should depend on an environment variable. Perhaps there's a way (an cmd line option?) to tell it to **also** pick the packages in ``requirements-optional.txt``

Reply:
I know this only work for travis for now I'm tryin to figure out what to do